addons/account/report/account_invoice_report.py

```diff
@@ -120,6 +129,7 @@ def _from(self):
                 LEFT JOIN uom_uom uom_template ON uom_template.id = template.uom_id
                 INNER JOIN account_move move ON move.id = line.move_id
                 LEFT JOIN res_partner commercial_partner ON commercial_partner.id = move.commercial_partner_id
+                LEFT JOIN ir_property product_standard_price ON product_standard_price.company_id = line.company_id AND product_standard_price.res_id = CONCAT('product.product,', product.id) AND product_standard_price.name = 'standard_price'
```

Comment:
Maybe add some indentation here + not 100% sure whether we can simplify this or not. 
I'll let the final reviewer enlighten us :slightly_smiling_face:

Reply:
I'll try to indent it somehow, matching the already existing indentation.
In terms of simplification, after having a second look, I think I don't need the company_id, since the `product.id` will already be unique. So I'll remove that one.